services/director-v2/tests/unit/with_dbs/test_modules_dynamic_sidecar_docker_service_specs.py

```diff
@@ -132,10 +133,10 @@ def expected_dynamic_sidecar_spec(
                     "key": "simcore/services/dynamic/3dviewer",
                     "node_uuid": "75c7f3f4-18f9-4678-8610-54a2ade78eaa",
                     "paths_mapping": {
-                        "inputs_path": "/tmp/inputs",
-                        "outputs_path": "/tmp/outputs",
-                        "state_exclude": ["/tmp/strip_me/*", "*.py"],
-                        "state_paths": ["/tmp/save_1", "/tmp_save_2"],
+                        "inputs_path": "/tmp/inputs",  # noqa: S108
+                        "outputs_path": "/tmp/outputs",  # noqa: S108
+                        "state_exclude": ["/tmp/strip_me/*", "*.py"],  # noqa: S108
+                        "state_paths": ["/tmp/save_1", "/tmp_save_2"],  # noqa: S108
                     },
                     "product_name": osparc_product_name,
                     "project_id": "dd1d04d9-d704-4f7e-8f0f-1ca60cc771fe",
```

Comment:
I guess in this whole section one could make heavy use of `faker`, maybe for another time

Reply:
I agree to both.